tests/test_phonons_cli.py

```diff
@@ -59,7 +60,7 @@ def test_phonons():
 
         has_eigenvectors = False
         has_velocity = False
-        with open(bands_path, encoding="utf8") as file:
```

Comment:
If this is a `yaml` file, do we want to just parse it and check the properties?

Reply:
I wanted to avoid parsing the full file, since its a huge file (335100 lines), and `eigenvector`/`group_velocity` appear within the first 60 lines.

Originally we did check the yaml, but it slows the tests down quite noticeably, and I don't think I found a way to partially parse yaml particularly nicely.